src/escpos/printer/cups.py

```diff
@@ -78,7 +84,7 @@ def is_usable() -> bool:
         return is_usable()
 
     @dependency_pycups
-    def __init__(self, printer_name=None, *args, **kwargs):
+    def __init__(self, printer_name: str = "", *args, **kwargs):
```

Comment:
You can remove `:type printer_name: ...` from the docstring if you have set it with a type annotation

Reply:
Done.
Resolving.